AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsPrivate.h

```diff
@@ -21,6 +21,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property(nonatomic, nullable) id<MSAnalyticsDelegate> delegate;
 
+@property(nonatomic) NSUInteger flushInterval;
+
+@property(nonatomic) MSChannelUnitConfiguration *channelUnitConfiguration;
```

Comment:
Just remove the readonly from `MSServiceCommon.h`, we don't need to redefine it here.